tools-reference/head-and-tail/text.xml

```diff
@@ -69,6 +69,18 @@ sed -n -e '1,5{ s/foo/bar/g ; p }' input.txt &gt; output.txt
 # on line 5
 sed -n -e 's/foo/bar/gp ; 5q' input.txt &gt; output.txt
 
+# bad: replace 'foo' with 'bar', and drop first five lines
+# (this one is from a real example!)
+sed -e 's/foo/bar/' input.txt &gt; tmp.txt
+tail -n $(($(wc -l tmp.txt | awk '{print $1}') - 5)) tmp.txt &gt; output.txt
+rm -f tmp.txt
+
+# better: let tail count lines from the beginning of the file
```

Comment:
there is an ambiguity in this ranking. I think `bad < good < better` makes more sense then `bad < better < good`. In other words, I think better is better than good. Or we can just get rid of better.

Reply:
Example moved to the first section. The main point is to illustrate that tail can start counting from beginning of the file.